### PR TITLE
Konem/hasura

### LINKED ISSUE #1301: Set up Hasura development environment (synchronize metadata)
Assigned to @kon3m with guidance from @ashwinkjoseph

### PR DESCRIPTION
**Pull request by kon3m**
## Description
enabled auto apply of migrations and metadata ,added seed data
## Release Checklist

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ ] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Security Checklist

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

### CODE REVIEW

**Comment by shayantist** [comment]
@kon3m thanks for the PR! Would you be able to explain a bit in layman's terms what this PR does in terms of setting up Hasura?

**Comment by kon3m** [comment]
@schowdhury8 Sure, the docker-compose file that has been added helps in setting up the backend (postgres and hasura). All a developer needs to do is make sure they have docker and docker-compose installed. Once that is done they can start the service by typing `docker-compose up --build -d` . The hasura image that is being used here helps in automatically applying migrations and metadata instead of them doing it explicilty.

**Review by ethanstrominger — APPROVED** [review]
